build-package.sh

```diff
@@ -169,9 +169,25 @@ if [ "$(uname)" == "Darwin" ]; then
 fi
 
 # Generate jazzy docs (macOS)
-if [ "${docs}" == "true" ] && [ "$(uname)" == "Darwin" ]; then
-    sourceScript "${projectFolder}/Package-Builder/jazzy.sh"
+#if [ "${docs}" == "true" ] && [ "$(uname)" == "Darwin" ]; then
+#sourceScript "${projectFolder}/Package-Builder/jazzy.sh"
+#fi
+
+# Generate jazzy docs (macOS) where the 'documentation' tag exists on the issue label
+if [ "$(uname)" == "darwin"* ] && [ "${TRAVIS_PULL_REQUEST}" != "false" ]; then
+    if  [ -z "${GITHUB_USERNAME}" ] && [ -z "${GITHUB_PASSWORD}" ]; then
+        response= curl -s -X GET echo "https://${GITHUB_USERNAME}:${GITHUB_PASSWORD}@api.github.com/repos/${TRAVIS_REPO_SLUG}/issues/${TRAVIS_PULL_REQUEST}/labels" | sed 's/\\\\\//\//g' | sed 's/[{}]//g' | awk -v k="text" '{n=split($0,a,","); for (i=1; i<=n; i++) print a[i]}' | sed 's/\"\:\"/\|/g' | sed 's/[\,]/ /g' | sed 's/\"//g' | grep -w 'name'
```

Comment:
Maybe divide this line and add commenting to explain the `sed` and `grep` commands

Reply:
Link to github API doc?

Reply:
Has been commented now.